### DISCUSSION TITLE
Add the fleet server role on the receiver

### DISCUSSION THREAD

**Discussion by Wilkins00** [Security-Onion-Solutions/securityonion#13706] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

512

### Storage for /

100

### Storage for /nsm

1tb

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi security onion team,

I would like to know if is possible to add the fleet server role to the receiver, actuality I have a receiver which is only receive the Logstash log but not the heartbeat from the agent  , the heardbeat from the agent are causing my manager to have some CPU and disk usage peek , I would like to enable the fleet server on the receiver without adding a new fleet server because my manager is not currently giving my new server.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
You maybe able to get something like this to work, by adding the fleet server integration to your receiver node and adding the receiver under "fleet server hosts". Also, configuring the receiver to accept traffic over the fleet server ports. This isn't a supported configuration and the custom setup may not be as straightforward.

If you are having resource issues with the manager and you have a receiver setup already. You should consider setting elasticfleet > enable_manager_output [adv] to 'False' https://docs.securityonion.net/en/2.4/architecture.html#receiver-node that way your manager stops taking in logs from agents and instead the receiver(s) take all logs.

**Comment by Wilkins00** [comment]
Hi reyesj2,
this should be in the pull requests session, sometimes we need the receiver to handle the fleet heartbeet and the logstash log.
